### DISCUSSION TITLE
VirtualDesktop-OpenXR

### DISCUSSION THREAD

**Discussion by Swinec** [alvr-org/ALVR#1868] [General]
Hey guys! Matthieu Bucchianeri make a openxr fork for wirelsess devices, buuuut its for propriental shitty VD.
It's a chance to make VR TOTALY FREE. No random bug update from Gaben, no crooked  streamer software from china/meta developers.
link - https://github.com/mbucchia/VirtualDesktop-OpenXR

How do I imagine it - vr-devices == monitor+keyboard. Why we must install shop-snitch software on PC for using regular hardware?!

Viva La RESISTANCE!

**Comment by zmerp** [comment]
Interesting project. Currently there is some initial effort to port ALVR to Monado, but there is no ETA, and it's still far off, and it's going to be Linux-only at first.
